### PR TITLE
add new topo and lldp testcases fit it

### PR DESCRIPTION
**Pull request by tiejunzhang-ms**
### Description of PR
Add one new topo to support cisco kvm as neighbor and LLDP testcases to fit this topo 

Summary:
Fixes # (issue)
1. add new topo to support cisco kvm as neighbor
2. add LLDP testcases to run based on this topo

### Type of change

- [ ] Bug fix
- [X] Testbed and Framework(new/improvement)
- [X] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
1. to support cisco kvm as dut neighbor
2. to make LLDP testcases run smoothly based on cisco kvm neighbor
#### How did you do it?
1. add new topo to support cisco neighbor

#### How did you verify/test it?
1. verify existing LLDP testcases based no new topo
2. verify new LLDP testcases based on new topo

#### Any platform specific information?
N/A
#### Supported testbed topology if it's a new test case?
N/A

### CODE REVIEW

File: ansible/library/lldp_facts.py [hunk 1]
```diff
@@ -90,7 +90,8 @@ def __init__(self,dotprefix=False):
             dp = ""
 
         # From IF-MIB
-        self.if_descr       = dp + ".0.8802.1.1.2.1.3.7.1.3"
+        #ifdescr is common support, replace the lldpportid
+        self.if_descr       = dp + ".3.6.1.2.1.2.2.1.2"
```
Comment (wangxin) [inline ansible/library/lldp_facts.py:94]: Will this change affect the existing LLDP testing?

**Comment by wangxin** [comment]
@tiejunzhang-ms Can you address the pre-commit issues for the new file you added?  `tests/wan/lldp/test_wan_lldp.py`

File: ansible/library/lldp_facts.py (hunk 1 above)
Comment (tiejunzhang-ms) [reply ansible/library/lldp_facts.py:94]: There is not any side effective with existing LLDP case. The main purpose is get interfaces collection from dut, and use these data for sanity check. I have double confirmed this based on running existing testcases.

**Comment by tiejunzhang-ms** [comment]
> @tiejunzhang-ms Can you address the pre-commit issues for the new file you added? `tests/wan/lldp/test_wan_lldp.py`

will do, thanks for comments

**Review by wangxin — APPROVED** [review]

**Pull request merged**